osh/hub/settings_local.py

```diff
@@ -29,7 +29,7 @@
         'HOST': 'db',
         'PORT': '5432',
         'TEST': {
-                'NAME': 'openscanhub-test',
```

Comment:
That was introduced by the linter. Fixed

Reply:
Hmm, that it weird.  `pre-commit` should not reformat any lines that were not edited.

Reply:
On PyCharm, if you click on "Reformat the file" to fix some suggestion, it will automatically format the whole document, also even for lines that were not edited.
<img width="438" alt="Screenshot 2024-01-24 at 13 56 10" src="https://github.com/openscanhub/openscanhub/assets/124347725/ac434880-4438-4063-b056-39aebc8e22cb">